spec/services/citizen_email_service_spec.rb

```diff
@@ -14,6 +15,7 @@
         .with(application.id, 'test@example.com', citizen_url, 'John Doe')
         .and_return(message_delivery)
       allow(message_delivery).to receive(:deliver_later)
+      allow(application).to receive(:generate_secure_id).and_return(secure_id)
```

Comment:
shouldn't it be `expect` here instead of `allow` (and the above line as well)?
I tend to use `allow` when stubbing behaviour in the `before` blocks.
And `expect` in the specs.